app/services/atmosphere/appliance_vms_manager.rb

```diff
@@ -120,6 +120,14 @@ def appliance_satisfied(vm)
       appliance.state = :satisfied
       updater.update(new_vm: vm)
       @tags_manager_class.new(vm).execute
+      register_in_cep_engine(vm)
+    end
+
+    def register_in_cep_engine(vm)
```

Comment:
Unused method argument - `vm`. If it's necessary, use `_` or `_vm` as an argument name to indicate that it won't be used. You can also write as `register_in_cep_engine(*)` if you want the method to accept any arguments but don't care about them.